lib/Screens/AboutUs.dart

```diff
@@ -1,10 +1,355 @@
+import 'dart:async';
+import 'dart:convert';
+
 import 'package:flutter/material.dart';
+import 'package:flutter_spinkit/flutter_spinkit.dart';
+import 'package:fluttertoast/fluttertoast.dart';
+import 'package:http/http.dart' as http;
+import 'package:url_launcher/url_launcher.dart';
+
+class AboutUs extends StatefulWidget {
+  @override
+  _AboutUsState createState() => _AboutUsState();
+}
+
+class _AboutUsState extends State<AboutUs> {
+  final double spacing = 40;
+  List jsonContributors = [];
+  String _about_this_app =
+      "Chapter3 is an app specifically made for renting books. It displays collections of top-selling books as well as the option to have your taste curated and matched with a book. Here customers can filter and select books based on genre, after which placing immediate or deferred orders with the option to cancel.";
+
+  Future<void> _fetchContributors() async {
+    const contributorsAPIUrl =
+        'https://api.github.com/repos/Saransh-cpp/BookRentApp-Chapter3/contributors';
+
+    final response = await http.get(Uri.parse(contributorsAPIUrl));
+
+    if (response.statusCode == 200) {
+      setState(() {
+        jsonContributors = jsonDecode(response.body) as List;
+      });
+    } else {
+      Fluttertoast.showToast(
+        msg: "Failed to load Contributors.",
+        toastLength: Toast.LENGTH_SHORT,
+        gravity: ToastGravity.BOTTOM,
+      );
+    }
+  }
+
+  bool _isLoading = true;
+
+  @override
+  void initState() {
+    super.initState();
+    _fetchContributors();
+    Future.delayed(const Duration(seconds: 2), () {
+      setState(() {
+        _isLoading = false;
+      });
+    });
+  }
 
-class AboutUs extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return Scaffold(
+      body: _isLoading
+          ? const SpinKitRotatingCircle(
+              color: Colors.blue,
+            )
+          : SafeArea(
```

Comment:
```suggestion
      body: SafeArea(
```

As we don't want the timer now.